### PR TITLE
Add Brazilian Portuguese locale

### PR DESCRIPTION
**Pull request by lincolnbrito**

### CODE REVIEW

File: Locale/pt_br/LC_MESSAGES/debug_kit.po [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+        "Content-Transfer-Encoding: 8bit\n"
+        "Plural-Forms: nplurals=INTEGER; plural=EXPRESSION;\n"
+
+#: controllers/components/toolbar.php:91
+msgid   "Component initialization and startup"
+msgstr  "Inicialização de componente"
+
+#: controllers/components/toolbar.php:140
+msgid   "Controller Action"
+msgstr  ""
```
Comment (markstory) [inline Locale/pt_br/LC_MESSAGES/debug_kit.po]: These should keep their english text until a translation is ready.

File: Locale/pt_br/LC_MESSAGES/debug_kit.po [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+msgid   "Graph"
+msgstr  "Gráfico"
+
+#: views/elements/variables_panel.ctp:21
+msgid   "View Variables"
+msgstr  "Visualizar Variáveis"
+
+#: views/helpers/simple_graph.php:79
+msgid   "Starting %sms into the request, taking %sms"
+msgstr  "Começando %ms na requisição, levando %ms"
```
Comment (jrbasso) [inline Locale/pt_br/LC_MESSAGES/debug_kit.po]: This should be `%sms` instead of `%ms` in both cases.

File: Locale/pt_br/LC_MESSAGES/debug_kit.po [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+msgid   ""
+msgstr  "Project-Id-Version: PROJECT VERSION\n"
+        "POT-Creation-Date: 2009-05-27 09:47+0200\n"
+        "PO-Revision-Date: YYYY-mm-DD HH:MM+ZZZZ\n"
+        "Last-Translator: Lincoln Brito <lincoln.sbrito@gmail.com>\n"
+        "Language-Team: LANGUAGE <EMAIL@ADDRESS>\n"
+        "MIME-Version: 1.0\n"
+        "Content-Type: text/plain; charset=UTF-8\n"
+        "Content-Transfer-Encoding: 8bit\n"
+        "Plural-Forms: nplurals=INTEGER; plural=EXPRESSION;\n"
```
Comment (jrbasso) [inline Locale/pt_br/LC_MESSAGES/debug_kit.po]: This needs to be `"Plural-Forms: nplurals=2; plural=n>1;\n"`

File: Locale/pt_br/LC_MESSAGES/debug_kit.po [hunk 4]
```diff
@@ -0,0 +1,135 @@
+# LANGUAGE translation of CakePHP Application
+# Copyright YEAR NAME <EMAIL@ADDRESS>
+# No version information was available in the source files.
+#
+#, fuzzy
+msgid   ""
+msgstr  "Project-Id-Version: PROJECT VERSION\n"
+        "POT-Creation-Date: 2009-05-27 09:47+0200\n"
```
Comment (jrbasso) [inline Locale/pt_br/LC_MESSAGES/debug_kit.po]: You probably can update to current date.

**Comment by jrbasso** [comment]
Portuguese translations looks right. :+1:

**Comment by dereuromark** [comment]
Sounds like this can be merged then.

**Comment by markstory** [comment]
Thanks @lincolnbrito!

**Comment by lincolnbrito** [comment]
Folks! Thanks for helping \0/

**Pull request merged**